python/srv.py

```diff
@@ -25,23 +25,34 @@
 
 logging.basicConfig(level=logging.INFO)
 
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    'database', help='path to database file, in-memory will be used if empty')
```

Comment:
Maybe we can move `database` to the end of the argument list and add `nargs='*'`so then it is no longer needed to supply empty string explicitly